src/sessions/softphone-session-handler.ts

```diff
@@ -511,7 +502,7 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
       return super.proceedWithSession(pendingSession);
     }
 
-    this.log('info', '`proceedWithSession` called with an active session and LA == 1. sending proceed via HTTP request', {
+    this.log('info', '`proceedWithSession` called with an active session and/or LA == 1. sending proceed via HTTP request', {
```

Comment:
Thoughts on adding "and/or" to this log? You can get here without being LA == 1.